AzmqCPack.cmake

```diff
@@ -0,0 +1,33 @@
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "AZMQ Boost Asio + ZeroMQ")
+set(CPACK_PACKAGE_DESCRIPTION
+"The azmq library provides Boost Asio style bindings for ZeroMQ
+
+This library is built on top of ZeroMQ's standard C interface and is
+intended to work well with C++ applications which use the Boost libraries
+in general, and Asio in particular.
+
+The main abstraction exposed by the library is azmq::socket which
+provides an Asio style socket interface to the underlying zeromq socket
+and interfaces with Asio's io_service().  The socket implementation
+participates in the io_service's reactor for asynchronous IO and
+may be freely mixed with other Asio socket types (raw TCP/UDP/Serial/etc.).")
+set(CPACK_PACKAGE_VERSION               "1.1.0")
```

Comment:
I will tag a release shortly with these changes, which will be version 1.0.3.  I will update this version to be in sync with the tag version